core/opendaq/device/src/device_info_impl.cpp

```diff
@@ -678,6 +681,28 @@ ErrCode DeviceInfoConfigImpl<TInterface, Interfaces...>::getServerCapabilities(I
     return OPENDAQ_SUCCESS;
 }
 
+template <typename TInterface, typename ... Interfaces>
+ErrCode DeviceInfoConfigImpl<TInterface, Interfaces...>::getConfigurationConnectionInfo(IPropertyObject** connectionInfo)
+{
+    BaseObjectPtr obj;
+    StringPtr str = "configurationConnectionInfo";
+    ErrCode err = this->getPropertyValue(str, &obj);
+    if (OPENDAQ_FAILED(err))
+        return err;
+    *connectionInfo = obj.asPtr<IPropertyObject>().detach();
+    return OPENDAQ_SUCCESS;
+}
+
+template <typename TInterface, typename ... Interfaces>
+PropertyObjectPtr DeviceInfoConfigImpl<TInterface, Interfaces...>::CreateDefaultConfigurationConnectionInfo()
+{
+    auto info = PropertyObject();
+    info.addProperty(StringProperty("protocolId", ""));
+    info.addProperty(StringProperty("address", ""));
```

Comment:
If we have the address we could also add the "port" to be complete and since its a property-object anyway